qiskit_nature/transformers/active_space_transformer.py

```diff
@@ -56,31 +56,66 @@ class ActiveSpaceTransformer(BaseTransformer):
           `active_orbitals`. This allows selecting an active space which is not placed around the
           Fermi level as described in the first case, above. When using this keyword argument, the
           following criteria must be met *in addition* to the ones listed above:
-              1. the length of `active_orbitals` must be equal to `num_orbitals`.
+              1. the length of `active_orbitals` must be equal to `num_molecular_orbitals`.
               2. the sum of electrons present in `active_orbitals` must be equal to `num_electrons`.
 
     References:
         - *M. Rossmannek, P. Barkoutsos, P. Ollitrault, and I. Tavernelli, arXiv:2009.01872
           (2020).*
     """
 
-    def __init__(self, num_electrons: int, num_orbitals: int, num_alpha: Optional[int] = None,
-                 active_orbitals: Optional[List[int]] = None):
+    def __init__(self,
+                 num_electrons: Optional[int] = None,
+                 num_molecular_orbitals: Optional[int] = None,
```

Comment:
So if say I have a triplet where the active space comprises 6 electrons. I say num_electrons is 6 and num_alpha is 2 as 2 of these are unpaired with a beta? So two fully occupied and two half occupied orbitals.

Reply:
No it's a little bit different than that. But I am wondering whether it would be easier (and also more consistent) if I just switched this to also take `Tuple[int, int]` for `num_electrons`. This is what everyone is used to in Qiskit anyways.
For simplicity, I can also make it a `Union[int, Tuple[int, int]]` which will assume singlet-spin.
I will also give this a thought tomorrow :+1: